### DISCUSSION TITLE
Welcome to dhaqsoPay Discussions!

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#1] [Announcements]
Here’s an updated version, including the Zoom meetings:

---

## 👋 Welcome to dhaqsoPay Discussions!

We're using Discussions as a place to connect with the **dhaqsoPay** community, share ideas, and collaborate on building this open-source payment testing toolkit together. Whether you're a student from Dugsiiye or a developer from elsewhere, we encourage you to join in and contribute!

Here’s what we hope to achieve in this space:
  * Ask questions related to the project or payment testing in general.
  * Share ideas and suggestions for improving **dhaqsoPay**.
  * Collaborate with other members of the community to solve issues.
  * Participate in discussions about upcoming features and updates.
  * Be open-minded and respectful. This is a community we are building together, and everyone’s input matters 💪.

To get started, introduce yourself in the comments below and let us know how you plan to contribute to **dhaqsoPay**.

### Weekly Zoom Meetings
We will be hosting **weekly live meetings on Zoom** to discuss the progress of the project, brainstorm ideas, and tackle any blockers. Links to join the Zoom meetings will be shared in our announcement section.

**Comment by shiinedev** [comment]
Waan dhawnahay bro

**Comment by A-M-Ahmed** [comment]
cool 🔥

**Comment by iftiiin** [comment]
Waa dhawnahy eng
